### ISSUE TITLE
Highlight reconstruction (guided laplacians): allow several instances. 

### ISSUE THREAD

**Issue by pedrorrodriguez** [aurelienpierreeng/ansel#132]
With the latest GL reconstruction speed has increased tremendously, allowing for a significant increase in iterations and thus, better results. 

However, for certain images (sunsets, for example), often the diameter of the reconstruction does not suit the whole image: what may look great for when all three channels are clipped may not look good for larger areas where only one channel is clipped. Inpaint a flat color only mitigates the issue somewhat. 

With previous methods, multiple instances didn't really make sense, but with GL, we could mask out several regions, and work them independently for best results. 

TLDR. Allow for multiple instances of the module so different areas can be isolated and treated with different settings.

**Comment by aurelienpierre** [comment]
Enabling instances and masking hastf the potential of breaking histories and compatibility

**Comment by pedrorrodriguez** [comment]
Perhaps my previous issue of increasing number of iterations for smaller diameters will take care of it. Let's see.

**Comment by pedrorrodriguez** [comment]
Btw, masking is already enabled, just not instances

**Comment by pedrorrodriguez** [comment]
What's the status on this? Should I close the issue?

**Comment by Jiyone** [comment]
No don't close it, I believe this will be helpful too, that's an option I would like to have too.
The issue is that the reconstruction applies on all HL issues on the pictures instead of adapt for each case.

**Comment by aurelienpierre** [comment]
Done.
